tds/src/main/java/thredds/server/config/TdsContext.java

```diff
@@ -233,6 +233,16 @@ public void afterPropertiesSet()
             }
         }
 
+        // jupyter notebooks dir

```

Comment:
This still needs a way to handle enable/disabling this feature (other than deleting the notebook folder from the content directory). We could treat '/notebook/' as a service and allow/disallow it similarly to the other services?

Reply:
Yes, given that it's a service, we should allow/disallow it in threddsConfig.xml. However, by default, I think it should be enabled.

Reply:
Notebook viewers are now treated as a service BUT the point at which allowed services are finalized is pretty late in the startup process, i.e., after the content directory is populated and after viewers are registered, so we always register a `JupyterNotebookViewer` and plop a `notebooks` folder in the content directory, and then we just don't use them if the service is disabled. Not really a problem, just a note.